src/uwtools/config/formats/yaml.py

```diff
@@ -157,13 +156,10 @@ def _yaml_include(self, loader: yaml.Loader, node: yaml.SequenceNode) -> dict:
     @property
     def _yaml_loader(self) -> type[yaml.SafeLoader]:
         """
-        The loader, with appropriate constructors added.
+        A loader with all UW constructors added.
         """
-        loader = yaml.SafeLoader
+        loader = uw_yaml_loader()
```

Comment:
The basic loader, handling tags `!bool`, `!datetime`, `!float`, and `!int`, has been factored out to `uwtools.config.support` so that it can be used by code outside `YAMLConfig`. This method obtains that loader, then adds `!include` support. It seemed far from straightforward to factor `!include` support out of `YAMLConfig`, and I think there are open questions about how the `!include` tag would interact with other features (e.g. an `!include`-tagged value that loads values tagged with other tags like `!int` et al, how these interact with Jinja2 value rendering, etc.) I think we'll have to cross that bridge when we come to it.

Reply:
I did a basic test involving tagged values and `!include` and things seem to work as I'd expect:

`numbers.yaml`
```
e_plus_half_pi: !float '{{ values.e + half_pi }}'
half_pi: !float '{{ values.pi / 2 }}'
values: !include [e.yaml, pi.yaml]
```
`e.yaml`
```
e: 2.718
```
`pi.yaml`
```
pi: 3.141
```
And
```
$ uw config realize --input-file numbers.yaml --output-format yaml
e_plus_half_pi: 4.2885
half_pi: 1.5705
values:
  e: 2.718
  pi: 3.141
```

Reply:
I'm not sure this is consistent with what is documented. In the docs from this PR, I see:

```
$ uw config realize --input-file numbers.yaml --output-format yaml
values:
  constants:
    pi: 3.141
```

As the result from `values: !include [e.yaml, pi.yaml]`.

Reply:
I see what you mean, and it took me a minute, but I think it's consistent:

The situation in the documented case is that, under `values:`, there's a repeated `constants:` key in both `e.yaml` and `pi.yaml`, so `constants: e: 2.718` gets clobbered by `constants: pi: 3.141`, leaving only the latter.

The situation in my comment above is that there is no repeated key under `values:`, so `e` and `pi` can coexist.